custom_components/tahoma/climate_devices/atlantic_electrical_heater.py

```diff
@@ -18,26 +18,26 @@
 
 COMMAND_SET_HEATING_LEVEL = "setHeatingLevel"
 
+CORE_ON_OFF_STATE = "core:OnOffState"
 IO_TARGET_HEATING_LEVEL_STATE = "io:TargetHeatingLevelState"
 
-PRESET_FREEZE = "Freeze"
-PRESET_FROST_PROTECTION = "frostprotection"
-PRESET_OFF = "off"
+PRESET_FREEZE = "freeze"
```

Comment:
Why did you rename "frost protection" to "freeze"? Frostprotection seems well named but I could be wrong.
In french, the function is named "hors-gel" and it will just set the temperature of the heater to 7°C to avoid any freezing in the house.

Although, the following presets are needed too in the Atlantic_Electrical_Heater device : 
PRESET_COMFORT1 = "comfort-1"
PRESET_COMFORT2 = "comfort-2"